SQuAD/train/dawn_qanet_1tpu.json

```diff
@@ -6,6 +6,7 @@
     "model": "QANet",
     "hardware": "1 TPUv2",
     "timestamp": "2018-04-20",
+    "codeURL": "https://github.com/tensorflow/tpu/tree/master/models/experimental/qanet",
```

Comment:
Please include a commit hash. You can either update the `codeURL` field or add a separate `commitHash` field.

Reply:
Added commit hash field.